Resources/Locale/en-US/emotes/emotes.ftl

```diff
@@ -1,2 +1,7 @@
 ﻿emote-deathgasp = seizes up and falls limp, {POSS-ADJ($entity)} eyes dead and lifeless...
 silicon-emote-deathgasp =seizes up and falls limp...
```

Comment:
This is the silicon deathgasp that is used in the original IPC PR. Remove if you want to replace it

Reply:
I've actually been wanting to replace the Deathgasps for awhile, so it's fine.